exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -54,14 +54,18 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
+@TestInstance(Lifecycle.PER_CLASS)
 class TestKitTest {
 
   @RegisterExtension
-  TestKitExtension testKitExtension = new TestKitExtension(
-      TestKit.builder(EmulatedNodeType.VALIDATOR)
+  static TestKitExtension testKitExtension = new TestKitExtension(
```

Comment:
Shall we use the extension in the test of testkit + its builder?

Reply:
I think it's okay to use extension for most of the TestKit tests (not builder tests though).